SETUP_ARCH.md

```diff
@@ -0,0 +1,23 @@
+# Snowdrift setup instructions for Arch Linux
+
+Arch has the benefit of being very well-documented, so this document is
+very short!
+
+1.  [Install Haskell][1]. While you are at it, make sure you have the
+    `base-devel` group installed.
```

Comment:
What is "while you are it" ? Does that mean this can be checked while Haskell is installing? Maybe it should just be a note before step 1 instead of part of step 1?